docs/documentation/make-first-prototype/use-components-2.md

```diff
@@ -32,6 +32,11 @@ Your component code should now look like this:
 
 Your page should now look like this:
 
-![Screenshot of the question page with a textarea](/public/images/docs/tutorial-textarea.png)
+<figure>
+
+![](/public/images/docs/tutorial-textarea.png)
```

Comment:
from this article:
https://www.scottohara.me/blog/2019/01/21/how-do-you-figure.html#a-figcaption-is-not-a-substitute-for-alt-text

"If an image is given an empty alt, then the figcaption is in effect describing nothing. And that doesn’t make much sense, does it?"

Reply:
I can see we have description for this in the spreadsheet, not alt, but I think it might be better as alt, then screen reader users can interact with the image element and know what it is

Reply:
sorry no it shouldnt be alt as is, as it's not descriptive

Reply:
So we need new content for the alt text?

Reply:
yep, suggest `Web page with the heading "What is your most impressive juggling trick", a textarea and continue button`